common/constants.go

```diff
@@ -33,16 +31,16 @@ const (
 var Rewound = uint64(0)
 
 var TIP2019Block = big.NewInt(1)
-var TIPSigning = big.NewInt(225000)
-var TIPRandomize = big.NewInt(225000)
-var TIPV2SwitchBlock = big.NewInt(7074000)
+var TIPSigning = big.NewInt(0)
+var TIPRandomize = big.NewInt(0)
+var TIPV2SwitchBlock = big.NewInt(0)
 
-var TIPNoHalvingMNReward = big.NewInt(429987) // hardfork no halving masternodes reward
-var BlackListHFNumber = uint64(225000)
-var TIPXDCX = big.NewInt(225000)
-var TIPXDCXLending = big.NewInt(225000)
-var TIPXDCXCancellationFee = big.NewInt(225000)
-var TIPXDCXCancellationFeeTestnet = big.NewInt(225000)
+var TIPNoHalvingMNReward = big.NewInt(0) // hardfork no halving masternodes reward
+var BlackListHFNumber = uint64(0)
+var TIPXDCX = big.NewInt(0)
+var TIPXDCXLending = big.NewInt(0)
+var TIPXDCXCancellationFee = big.NewInt(0)
+var TIPXDCXCancellationFeeTestnet = big.NewInt(0)
```

Comment:
I think we should be able to remove those constants right? No need to check them in the code anymore

Reply:
It will be too much change in the current state. And lots of files will get changed. I think we can just remain this as legacy code. Also merge conflict will be less if merge from `XDPoSChain`